static/js/util/profile_edit_test.js

```diff
@@ -145,10 +145,11 @@ describe('Profile Editing utility functions', () => {
     });
 
     let rerender = omitDay => {
+      let legend, space;  // eslint-disable-line no-unused-vars
       if (omitDay) {
-        [monthTextField, , yearTextField] = renderDateField(true);
+        [legend, space, monthTextField, , yearTextField] = renderDateField(true);
```

Comment:
As long as you're here, do you mind refactoring this function to use enzyme, if you feel like it? `renderDateField` is kind of a hacky workaround at the moment